tests/test_file_download.py

```diff
@@ -1226,6 +1227,7 @@ def test_large_file_http_path_error(self):
         "hf_transfer not installed, so skipping large file download with hf_transfer check.",
     )
     @patch("huggingface_hub.file_download.constants.HF_HUB_ENABLE_HF_TRANSFER", True)
+    @patch("huggingface_hub.file_download.constants.HF_HUB_DISABLE_XET", False)
```

Comment:
this is useful only when running the tests locally, but we should set it to `True` here instead to disable xet path for this specific test and correctly fallback to `http_get`
```suggestion
    @patch("huggingface_hub.file_download.constants.HF_HUB_DISABLE_XET", True)
```